### ISSUE TITLE
Check move/ability/item descriptions to see if their length fits.

### ISSUE THREAD

**Issue by AsparagusEduardo** [rh-hideout/pokeemerald-expansion#539]
I've been noticing a lot of moves, abilities and item descriptions which are going outside their respective windows. (Like the Shock Drive).
Writing this to remember to check all of them.

- [x] Items
- [x] Moves
- [x] Abilities

**Comment by LOuroboros** [comment]
I compiled a list of the items that need a revision, as well as other relevant tweaks.

**Items to fix**:

* Honey (description and sprite)
* Custap, Jaboca, Rowap, Kee, Maranga Berries (missing item data)
* Power Herb (description)
* Focus Sash (description)
* Zoom Lens (description)
* Wise Glasses (description)
* Light Clay (description)
* Icy, Smooth, Heat and Damp Rocks (description)
* Destiny Knot (description)
* Iron Ball (description)
* Float Stone (description)
* Zap Plate (description)
* Fist Plate (description)
* Mind Plate (description)
* Odd Keystone (missing item data)
* Griseous Orb (description)
* Shock Drive (description)
* Mega Bracelet (description)

**To do**:
* Add missing items
* Remove data related to TMs 51 to 100 and adjust the list of items accordingly.

I'll be handling descriptions and missing data. Will need to hear some feedback about the last 2 points, mainly the last one.
As discussed on the Discord server, it's pointless to have defined labels for TMs that are missing every single other piece of data, not to mention that 8 of them share internal IDs with the HMs which is silly as users need to take it into account when expanding TMs on their projects.
And that's not even considering that the item_expansion branch will not handle the task of expanding TMs either.
TLDR: As it stands, having defined TMs 51 to 100 is utterly pointless and can be confusing.
